docs/en/setup/ai-pipeline/README.md

```diff
@@ -0,0 +1,22 @@
+# AI Pipeline
+
+**Warning, this module is still in the ALPHA stage. This is not stable.**
+
+Pattern Recognition, Machine Learning(ML) and Artificial Intelligence(AI) are common technology to identify patterns in data. 
+This module provides a way to integrate these technologies in a standardized way about shipping the data from OAP kernel
+to 3rd party.
+
+From the industry practice, Pattern Recognition, Machine Learning(ML) and Artificial Intelligence(AI) are always overestimated,
+they are good at many things but have to run in a clear context.
+
+## How to enable
+
+ai-pipeline module is required to activate manually through `application.yml` or system environment variable `SW_AI_PIPELINE`. 
+
+```yaml
+ai-pipeline:
+  selector: ${SW_AI_PIPELINE:-}
+  default:
+    uriRecognitionServerAddr: ${SW_AI_PIPELINE_URI_RECOGNITION_SERVER_ADDR:0.0.0.0}
+    uriRecognitionServerPort: ${SW_AI_PIPELINE_URI_RECOGNITION_SERVER_PORT:17128}
```

Comment:
What's the rationale of using port `17128`? If there is no specific reason to use this port, I'd suggest using a port near to `11800 ~ 12800`, it's not a good practice to occupy a too wide port range for a system

Reply:
It is remote(skyapm r3 server address).